<launch>
    <!-- RT1 -->
	<node name="rosrt_rt1"          pkg="ros_start"  type="rosrt_rt1"/>
	<node name="rt1_con"            pkg="ros_start"  type="rt1_con"/>
	<node name="rt1_handle"		pkg="ros_start"  type="rt1_handle"/>
</launch>
